src/app/home/home.page.ts

```diff
@@ -15,14 +14,17 @@ interface DeviceHardware {
 })
 export class HomePage {
   isAppInstalled: any;
-  hardwareAvailiability: DeviceHardware = {
-    isBletoothTurnOn: false,
-    isGPSOn:false
+  hardwareAvailiability: Hardware = {
+    appAvailability: false,
```

Comment:
Logically app does not comes under hardware category. so we can move 'appAvailability' parameter outside object, or we can just rename 'hardwareAvailability' object to 'hardwareSoftwareAvailability'.